### ISSUE TITLE
Add nightly CI builds to 6.0 and 5.4 branch

### ISSUE THREAD

**Issue by seisman** [GenericMappingTools/gmt#1028]
**Description of the problem**

@leouieda Currently, we only run nightly builds (i.e. running tests, build documentations) on the master branch. We should do the same to the 6.0 and 5.4 branch.

**Comment by leouieda** [comment]
I don't know. It might get tricky having diverging CI configs for different branches. Could cause problems when merging changes. But we could try loosening the restrictions on which branches to build.

**Comment by seisman** [comment]
The master and 6.0 branches share the same CI configs. I think the only thing need to be done is to enable the cron jobs through the travis and azure admin panels.

**Comment by leouieda** [comment]
:+1: sounds good to me

**Comment by joa-quim** [comment]
GMT5 received only a couple of bug fixes in 2 years. I don't really see the point of wasting electricity running tests on it.

**Comment by seisman** [comment]
OK. At least we should run 6.0 branch nightly.

**Comment by PaulWessel** [comment]
Can it be done less frequently than nightly, e.g., monthly?

**Comment by seisman** [comment]
OK. I have enable nightly builds for 6.0 branch and monthly builds for 5.4 branch on Travis. I don't have permission to modify azure pipelines settings. @leouieda can you help with this.

In 5.4 branch, we now have many failing tests, due to remote file issues:
```
gmtwhich: Libcurl Error: Remote file not found
gmtwhich: Failed to remove /home/travis/.gmt/cache/wus_gps_final.txt! [remove error: No such file or directory]
gmtwhich: Could not even remove file /home/travis/.gmt/cache/wus_gps_final.txt
```

See https://travis-ci.org/GenericMappingTools/gmt/jobs/551975503 for build logs.

**Comment by leouieda** [comment]
@seisman do you have a Microsoft account? If you give me your login email I can add you as admin.

**Comment by seisman** [comment]
@leouieda seisman at outlook.com. Thanks.

**Comment by leouieda** [comment]
@seisman done

**Comment by seisman** [comment]
Closed by #1034.
